controllers/src/main/scala/cromwell/pipeline/controller/ProjectSearchController.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{ ExceptionHandler, Route }
+import cromwell.pipeline.controller.ProjectSearchController.projectSearchExceptionHandler
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.datastorage.dto.auth.AccessTokenContent
+import cromwell.pipeline.service.ProjectSearchService
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+
+class ProjectSearchController(projectSearchService: ProjectSearchService) {
+
+  private def searchProjects(implicit accessToken: AccessTokenContent): Route = post {
+    entity(as[ProjectSearchRequest]) { request =>
+      complete(projectSearchService.searchProjects(request, accessToken.userId))
```

Comment:
The scheme of the returned object here does not correspond to what's required by ticket. It should be 
```
{
  data: ["here go projects"]
}
```
and not
```json
["here go projects"]
```
This is required to make API extensible and be able to add more metainformation in the future (like filter id or pagination information)

Reply:
Fixed